app/views/contacts/edit.html.erb

```diff
@@ -1,6 +1,7 @@
-<% content_for :pre_content_nav do %>
-  <%= govuk_back_link(href: project_contacts_path(@project)) %>
-<% end %>
+
```

Comment:
Can we remove the whitespace changes here? They add noise to the PR and the commit history